doc/nrf/ug_nrf7002.rst

```diff
@@ -0,0 +1,275 @@
+
+1. Build the Non-Secure |NCS| application for the application core using ``-DBOARD=nrf7002dk_nrf5340_cpuapp_ns``.
+   To invoke the building of TF-M, the |NCS| build system requires the Kconfig option ``BUILD_WITH_TFM`` to be enabled, which is done by default when building |NCS| as a Non-Secure application.
+
+   The |NCS| build system performs the following steps automatically:
+
+      a. Build the Non-Secure firmware image as a regular |NCS| application.
+      #. Build a TF-M (secure) firmware image.
+      #. Merge the output image binaries.
+      #. Optionally build a bootloader image (MCUboot).
+
+   .. note::
+      Depending on the TF-M configuration, an application DTS overlay can be required, to adjust the Non-Secure image flash memory partition and SRAM starting address and sizes.
+
+2. Build the application firmware for the network core using ``-DBOARD=nrf7002dk_nrf5340_cpunet``.
+
+
+Building the Secure firmware using |NCS|
+-----------------------------------------
+
+To build the Secure and the Non-Secure firmware images using |NCS|, follow these steps:
+
+1. Build the Secure |NCS| application for the application core using ``-DBOARD=nrf7002dk_nrf5340_cpuapp``.
+   Also set ``CONFIG_TRUSTED_EXECUTION_SECURE=y`` and ``CONFIG_BUILD_WITH_TFM=n`` in the application project configuration file.
+2. Build the Non-Secure |NCS| application for the application core using ``-DBOARD=nrf7002dk_nrf5340_cpuapp_ns``.
+3. Merge the two binaries.
+4. Build the application firmware for the network core using ``-DBOARD=nrf7002dk_nrf5340_cpunet``.
+
+
+When building a Secure and a Non-Secure application for the nRF5340 application core, the Secure application has to set the IDAU (SPU) configuration to allow Non-Secure access to all CPU resources utilized by the Non-Secure application firmware.
+SPU configuration shall take place before jumping to the Non-Secure application.
+
+Building a Secure only application
+==================================
+
+Build the |NCS| app following the standard process (see :ref:`build_an_application` and :ref:`application_run`), using ``-DBOARD=nrf7002dk_nrf5340_cpuapp`` for the firmware running on the nRF5340 application core and ``-DBOARD=nrf7002dk_nrf5340_cpunet`` for the firmware running on the nRF5340 network core.
+
+Programming the firmware to the DK
+==================================
+
+Follow the instructions in the :ref:`nordic_segger` page to install and configure all the necessary software.
+Further information can be found in :ref:`nordic_segger_flashing`.
+Then you can build and flash applications as usual (:ref:`build_an_application` and :ref:`application_run` for more details).
+
+.. note::
+
+   The nRF5340 has a flash read-back protection feature.
+   When flash read-back protection is active, you will need to recover the chip before reflashing.
+   If you are flashing with :ref:`west <west-build-flash-debug>`, run this command for more details on the related ``--recover`` option:
```

Comment:
```suggestion
   When this feature is active, you need to recover the chip before reflashing.
   If you are flashing with :ref:`west <west-build-flash-debug>`, run this command for more details on the related ``--recover`` option:
```
Does running this command really give more details on the ``-- recover`` option?